index.js

```diff
@@ -77,6 +77,12 @@ Prober.prototype.notok = function notok() {
     }
 };
 
+Prober.prototype.setEnabled = function setEnabled(enabled) {
+    if (typeof enabled === 'boolean') {
```

Comment:
we can put two methods: `enable` and `disable`, and leave the interpretation one layer up.